configs/config_hg38.json

```diff
@@ -98,6 +98,11 @@
         {
             "annotate_ref": "/apps/bio/dependencies/wgs_somatic/hg38/refseq/RefseqAll_ncbiRefSeq_hg38_GenesGenepredictions.gtf"            
         },
+        "ascat":
+        {
+            "resources": "/apps/bio/dependencies/wgs_somatic/hg38/ascat",
+            "singularity": "/apps/bio/dependencies/wgs_somatic/hg38/ascat/ascat.sif"         
```

Comment:
Why not follow the structure of the config and add the path to the singularity where the other singularities are specified?

I see that one could want to change this so that the singularitues are grouped together with the configs of the corresponding tool but atm this is not the case...

Reply:
I read this in `/apps/bio/singularities/README `  from 2022

> This folder contains various images and dependencies.
>They should at some point be moved over to /apps/bio/dependencies
>
>Therefore, nothing new should be added here.

I also had various permission errors in the /apps/bio/singularities directory, but I guess that could be resolved